drm2/radeonkms/Makefile

```diff
@@ -110,7 +110,7 @@ SRCS	+=								\
 
 CFLAGS+= -I${.CURDIR:H}/../src/
 
-CFLAGS+=			-I${SRCTOP}/../src/dev/drm2/radeon
+CFLAGS+=			-I${.CURDIR:H:H}/../src/dev/drm2/radeon
```

Comment:
I'm not sure this is right. What are you trying to accomplish?

Reply:
`SRCTOP` should probably be changed to relative path. Could be missed change from when we moved it to ports. It might have worked while drm2 was still in base but fail now when it's removed if SRCTOP points to /usr/src/...